src/CMakeLists.txt

```diff
@@ -192,6 +196,8 @@ if(INTEL_SSE4_1)
   if (INTEL_SSE4_2)
     set(SSE4_FLAGS "${SSE4_FLAGS} -msse4.2")
   endif (INTEL_SSE4_2)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${SSE4_FLAGS}")
```

Comment:
@tchaikov Right. Should leave the choice to the user. I will roll back the change in src/CMakeLists.txt, and update erasure-code/jerasure/CMakeLists.txt